### PR TITLE
feat: use framework.ExpectEqual instead of should

### PR DESCRIPTION
**Pull request by Moriadry-zz**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

/kind cleanup
/sig testing
/assign @oomichi

**What this PR does / why we need it**:
use framework.ExpectEqual instead of should in e2e test

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

NONE
```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Review by draveness — APPROVED** [review]
/lgtm
/ok-to-test

Thanks for the cleanup

**Comment by Moriadry** [comment]
@draveness 
hello, I cannot figure why those two tests failed. Could help me check this ?

File: test/e2e_node/mirror_pod_test.go [hunk 1]
```diff
@@ -81,8 +81,8 @@ var _ = framework.KubeDescribe("MirrorPod", func() {
 			ginkgo.By("check the mirror pod container image is updated")
 			pod, err = f.ClientSet.CoreV1().Pods(ns).Get(mirrorPodName, metav1.GetOptions{})
 			gomega.Expect(err).ShouldNot(gomega.HaveOccurred())
-			gomega.Expect(len(pod.Spec.Containers)).Should(gomega.Equal(1))
-			gomega.Expect(pod.Spec.Containers[0].Image).Should(gomega.Equal(image))
+			framework.ExpectEqual(pod.Spec.Containers, 1)
```
Comment (oomichi) [inline test/e2e_node/mirror_pod_test.go]: We should not remove `len()`. That is the reason why the test job is failed now.

**Review by oomichi — CHANGES_REQUESTED** [review]
need to fix the test code.

File: test/e2e_node/mirror_pod_test.go (hunk 1 above)
Comment (Moriadry) [reply test/e2e_node/mirror_pod_test.go]: Ooh, I didn’t noticed the failure, sorry for that.! Will fix it ASAP.

**Comment by oomichi** [comment]
The change itself is good for me now. Thanks for updating.
But we need to rebase this for the latest master, please.

**Pull request closed without merging**